homeassistant/components/plugwise/gateway.py

```diff
@@ -26,6 +26,7 @@
     COORDINATOR,
     DEFAULT_PORT,
     DEFAULT_SCAN_INTERVAL,
+    DEFAULT_TIMEOUT,
```

Comment:
shouldnt we be importing CONF_USERNAME as well and usingit for Smile with default since it was added to config flow?

Reply:
Yes, we should, thanks!

Reply:
Latest commit should fix tests (adding `DEFAULT_USERNAME`).